src/components/application_manager/test/commands/mobile/subscribe_way_points_request_test.cc

```diff
@@ -98,7 +98,8 @@ TEST_F(SubscribeWayPointsRequestTest, Run_ApplicationIsNotRegistered) {
 
 TEST_F(SubscribeWayPointsRequestTest, Run_WayPointsSubscribedBefore_Ignored) {
   CommandPtr command(CreateCommand<SubscribeWayPointsRequest>(msg_));
-  ON_CALL(mock_app_manager_, application(kKey)).WillByDefault(Return(app_));
+  ON_CALL(mock_app_manager_, application(kConnectionKey))
+      .WillByDefault(Return(app_));
```

Comment:
@wolfylambova22, are we really don't expect this call?
Please check everywhere for this type of issue